arff_driver.py

```diff
@@ -0,0 +1,184 @@
+import os
+import random
+from scipy.io import wavfile
+from scipy.fftpack import fft,fftfreq   # Likely won't be needed
+from Andrew import Andrew
```

Comment:
We may want to comment these out until we've all merged in some working form of each of our classes

Reply:
Unless it isn't throwing any errors